i18n/spec/closureI18nExtractorSpec.js

```diff
@@ -75,7 +75,6 @@ describe('findLocaleId', function() {
   it('should not find localeId if data is missing', function() {
     expect(findLocaleId('', 'num')).toBeUndefined();
     expect(findLocaleId('aa', 'datetime')).toBeUndefined();
-    expect(findLocaleId('aa', 'randomType')).toBeUndefined();
```

Comment:
This throws an "unknown type in findLocaleId: randomType" error. This what the next test verifies.
I assume that at some point someone decided to change the implementation, so that unknown types cause an error (vs returning `undefined`), but didn't update this test.

Maybe we should run these tests on CI.